src/test/java/blackjack/domain/GameResultTest.java

```diff
@@ -0,0 +1,103 @@
+package blackjack.domain;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+@Nested
+class GameResultTest {
+
+    @Nested
+    @DisplayName("승패 결정 테스트")
+    class MatchResultTest {
+
+        Player player;
+        Dealer dealer;
+
+        @BeforeEach
+        void initParticipants() {
+            player = new Player("sana");
+            dealer = new Dealer();
+        }
+
+        @Test
+        @DisplayName("블랙잭 vs 블랙잭이면 무승부를 반환한다.")
+        void matchBlackJackVsBlackJack() {
+            List<Card> blackjackCards1 = List.of(
+                new Card(Suit.SPADE, Denomination.TEN),
+                new Card(Suit.CLUB, Denomination.ACE)
+            );
+            player.addCards(blackjackCards1.get(0), blackjackCards1.get(1));
+
+            List<Card> blackjackCards2 = List.of(
+                new Card(Suit.SPADE, Denomination.KING),
+                new Card(Suit.CLUB, Denomination.ACE)
+            );
+            dealer.addCards(blackjackCards2.get(0), blackjackCards2.get(1));
+
+            assertThat(GameResult.playerResultFrom(dealer, player)).isEqualTo(GameResult.DRAW);
+        }
+
+        @Test
+        @DisplayName("버스트 vs 버스트이면 플레이어는 패배를 반환한다.")
+        void matchBustVsBust() {
+            List<Card> bustCards1 = List.of(
+                new Card(Suit.SPADE, Denomination.TEN),
+                new Card(Suit.CLUB, Denomination.KING),
+                new Card(Suit.CLUB, Denomination.THREE)
+            );
+            player.addCards(bustCards1.get(0), bustCards1.get(1), bustCards1.get(2));
```

Comment:
blackjackCards1를 만들어서 index로 요소에 접근해야 할까요? 가변 인자를 허용하셨기 때문에 아래와 같이 addCards 인자에 바로 넣어도 좋아보여요

```suggestion
            player.addCards(
                    new Card(Suit.SPADE, Denomination.TEN),
                    new Card(Suit.CLUB, Denomination.KING),
                    new Card(Suit.CLUB, Denomination.THREE)
            );
```

Reply:
카드의 합의 상태를 리스트의 이름으로 표현하려고 했는데 오히려 가독성이 나빠진 것 같습니다. 모두 바로 카드를 생성하여 파라미터로 전달하도록 변경했습니다!